source/track_thick.f90

```diff
@@ -656,7 +657,7 @@ subroutine thck4d(nthinerr)
         end if
         if(fluka_inside) then
           if(fluka_debug) then
-            write(lout,*) '[Fluka] Skipping lattice element at ',i
+            write(lout,"(a,i0)") "FLUKA> Skipping lattice element at ",i
```

Comment:
Make sure to give a heads-up to @amereghe about this change, in case of scripts that are `grep`ing.